ui/revenuecatui/src/androidTest/kotlin/com/revenuecat/purchases/ui/revenuecatui/PaywallUITest.kt

```diff
@@ -0,0 +1,63 @@
+package com.revenuecat.purchases.ui.revenuecatui
+
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.compose.ui.test.onNodeWithTag
+import androidx.compose.ui.test.performClick
+import com.revenuecat.purchases.PackageType
+import com.revenuecat.purchases.ui.revenuecatui.composables.PURCHASE_BUTTON_TAG
+import com.revenuecat.purchases.ui.revenuecatui.data.testdata.TestData
+import com.revenuecat.purchases.ui.revenuecatui.mocks.FakeViewModel
+import com.revenuecat.purchases.ui.revenuecatui.templates.template2SelectButtonTestTag
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+
+@OptIn(ExperimentalPreviewRevenueCatUIPurchasesAPI::class)
+class PaywallUITest {
```

Comment:
I'm currently testing CircleCI's tools to run ui tests instead of firebase test lab... But that's basically a job predefined in the android module that runs all instrumentation tests in all modules. However, we need more customization to modify the API keys, so I'm testing that right now...

Reply:
Also, if you look at the orb source https://circleci.com/developer/orbs/orb/circleci/android#commands-start-emulator-and-run-tests, it will run `./gradlew connectedDebugAndroidTest` which in android means run whatever is in the `androidTest` folder which is the instrumented tests